src/rgw/setup/rgw.py

```diff
@@ -550,7 +550,9 @@ def _update_hax_endpoint_and_create_admin(conf: MappedConf):
                           f' endpoint {e}')
                 break
         if rgw_lock is True:
-            current_data_node = socket.gethostname().replace('server', 'data')
+            # To read data pod hostname of current node
```

Comment:
Avoid using pod of kubernetes term in CORTX code.

Reply:
Sure @ujjwalpl, will not use the term going forward.
Also, this patch is a workaround to unblock provisioner team